SheepDash/core/src/com/missionbit/game/states/Level2.java

```diff
@@ -4,12 +4,131 @@
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.graphics.g2d.SpriteBatch;
 import com.badlogic.gdx.math.Vector2;
-import com.badlogic.gdx.utils.Array;
+import com.missionbit.game.GameTutorial;
+import com.missionbit.game.sprites.Farmer;
+import com.missionbit.game.sprites.Sheep;
 
 /**
  * Created by missionbit on 6/28/17.
  */
 
-public class Level2 {
+public class Level2 extends State {
+    private Texture sky;
+    private Sheep sheep;
+    private Farmer farmer;
+    private Texture buildings;
+    private Texture ground;
+    private Vector2 groundPos1;
+    private Vector2 groundPos2;
+    private Vector2 skyPos;
+    private Vector2 buildingsPos;
```

Comment:
To solve the problem with the buildings, I'd recommend you draw a third one. 
Also, you can declare all positions in one line `private Vector2 buildingsPos1, buildingsPos2, buildingsPos3;`